src/kernel/kernel.c

```diff
@@ -15,6 +15,10 @@
 #define STACK_COOKIE   0xDEADBEEFBAADC0DE
 #define STACK_GUARD    64
 
+#define TDATA_SIZE ((uint32_t)(__tdata_end)-(uint32_t)(__tdata_start))
+#define TLS_SIZE ((uint32_t)(__tls_end)-(uint32_t)(__tls_base))
+#define TP_OFFSET 28672
```

Comment:
You could spell this 0x7000 so that it looks a bit less arbitrary

Reply:
Done in kernel_internal.h